<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-BSVQKOV" failures="1" tests="1" name="Default test" time="0.262" errors="0" timestamp="22 Jan 2020 03:01:42 GMT">
  <testcase classname="dbtest.DB_SelTest" name="dbConnection" time="0.234">
    <failure type="java.sql.SQLException" message="Client does not support authentication protocol requested by server; consider upgrading MySQL client">
      <![CDATA[java.sql.SQLException: Client does not support authentication protocol requested by server; consider upgrading MySQL client
at mysql.connector/com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2847)
at mysql.connector/com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:750)
at mysql.connector/com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3558)
at mysql.connector/com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1151)
at mysql.connector/com.mysql.jdbc.Connection.createNewIO(Connection.java:1768)
at mysql.connector/com.mysql.jdbc.Connection.<init>(Connection.java:430)
at mysql.connector/com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:268)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
at dbtest.DB_SelTest.dbConnection(DB_SelTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- dbConnection -->
</testsuite> <!-- Default test -->
